mergify_cli/tests/ci/paths_to_scopes/test_cli.py

```diff
@@ -0,0 +1,473 @@
+#
+#  Copyright © 2021-2024 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+import pathlib
+import subprocess
+import tempfile
+from unittest import mock
+
+import click
+import pytest
+import yaml
+
+from mergify_cli.ci.paths_to_scopes import cli
+
+
+def test_from_yaml_valid() -> None:
+    with tempfile.NamedTemporaryFile(
```

Comment:
I think pytest has fixtures for this: [tmp_path](https://docs.pytest.org/en/6.2.x/tmpdir.html)

Reply:
vibe coding is not yet ready..., I asked to remove all of this and it misses some. I should I have checked all tests.